scripts/west_commands/runners/xsdb.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Runner for flashing with xsdb CLI, the official programming
+   utility from AMD platforms.
+"""
+import argparse
+import os
+from runners.core import ZephyrBinaryRunner, RunnerCaps, RunnerConfig
+
+class XSDBBinaryRunner(ZephyrBinaryRunner):
+    def __init__(self, cfg: RunnerConfig, config=None, bitstream=None,
+            fsbl=None):
+        super(XSDBBinaryRunner, self).__init__(cfg)
+        self.elf_file = cfg.elf_file
+        if not config:
+            cfgfile_path = os.path.join(cfg.board_dir, 'support')
+            default = os.path.join(cfgfile_path, 'xsdb.cfg')
+            if os.path.exists(default):
+                config = default
+        self.xsdb_cfg_file = config
+        self.bitstream = bitstream
+        self.fsbl = fsbl
+
+    @classmethod
+    def name(cls):
+        return 'xsdb'
+
+    @classmethod
+    def capabilities(cls):
+        return RunnerCaps(flash_addr=True)
+
+    @classmethod
+    def do_add_parser(cls, parser):
+        parser.add_argument('--config', help='if given, override default config file')
+        parser.add_argument('--bitstream', help='path to the bitstream file')
+        parser.add_argument('--fsbl', help='path to the r5 fsbl elf file')
```

Comment:
I'd suggest removing "r5" from here, as this should work with e.g. Zynq 7000 series as well?

```suggestion
        parser.add_argument('--fsbl', help='path to the fsbl.elf file')
```

Reply:
Sure will remove